hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/FullScanPhysicalRule.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.opt.physical;
+
+import com.hazelcast.jet.sql.impl.opt.OptUtils;
+import com.hazelcast.jet.sql.impl.opt.logical.FullScanLogicalRel;
+import org.apache.calcite.plan.RelOptRule;
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.convert.ConverterRule;
+
+import static com.hazelcast.jet.sql.impl.opt.JetConventions.LOGICAL;
+import static com.hazelcast.jet.sql.impl.opt.JetConventions.PHYSICAL;
+
+final class FullScanPhysicalRule extends ConverterRule {
+
+    static final RelOptRule INSTANCE = new FullScanPhysicalRule();
+
+    private FullScanPhysicalRule() {
+        super(
+                FullScanLogicalRel.class, LOGICAL, PHYSICAL,
+                FullScanPhysicalRule.class.getSimpleName()
+        );
+    }
+
+    @Override
+    public RelNode convert(RelNode rel) {
+        FullScanLogicalRel logicalScan = (FullScanLogicalRel) rel;
+
+        return new FullScanPhysicalRel(
```

Comment:
Looks like Jet couldn't use indexes at the moment. Is this the case? If yes, then it should be relatively simple to refactor IMDG part (`IndexResolver`) a bit, to allow for index planning in Jet.

Reply:
This PR cannot read IMaps at all using Jet for now.